app/src/main/java/com/hedvig/app/util/featureflags/flags/HAnalyticsFeatureFlagProvider.kt

```diff
@@ -9,13 +9,14 @@ class HAnalyticsFeatureFlagProvider(
     //  Feature.MOVING_FLOW -> marketManager.market == Market.SE || marketManager.market == Market.NO
     //  Feature.CONNECT_PAYMENT_AT_SIGN -> marketManager.market == Market.NO || marketManager.market == Market.DK
     override suspend fun isFeatureEnabled(feature: Feature) = when (feature) {
+        Feature.CONNECT_PAYMENT_AT_SIGN -> hAnalytics.postOnboardingShowPaymentStep()
         Feature.EXTERNAL_DATA_COLLECTION -> hAnalytics.allowExternalDataCollection()
         Feature.FRANCE_MARKET -> hAnalytics.frenchMarket()
         Feature.KEY_GEAR -> hAnalytics.keyGear()
         Feature.MOVING_FLOW -> hAnalytics.movingFlow()
-        Feature.CONNECT_PAYMENT_AT_SIGN -> hAnalytics.postOnboardingShowPaymentStep()
-        Feature.UPDATE_NECESSARY -> hAnalytics.updateNecessary()
-        Feature.REFERRAL_CAMPAIGN -> hAnalytics.foreverFebruaryCampaign()
         Feature.QUOTE_CART -> hAnalytics.useQuoteCart()
+        Feature.REFERRAL_CAMPAIGN -> hAnalytics.foreverFebruaryCampaign()
+        Feature.SHOW_CHARITY -> false // TODO hAnalytics.qasa()
```

Comment:
```suggestion
        Feature.SHOW_CHARITY -> TODO()
```

This way, we won't accidentally release this code, as there will be a certain crash on the profile tab.

Reply:
Yeah very good idea. Will fix right away!